packages/view-types/spatial/src/SpatialOptions.js

```diff
@@ -94,6 +100,20 @@ export default function SpatialOptions(props) {
           use3d={use3d}
         />
       ) : null}
+      <TableRow>
+        <TableCell className={classes.labelCell}>
+          Disable Tooltip
+        </TableCell>
+        <TableCell className={classes.inputCell}>
+          <Checkbox
+            className={classes.checkbox}
+            value={tooltipDisabled}
+            onChange={handleTooltipVisibilityChange}
+            name="gene-expression-colormap-option-toltip-visibility"
+            color="default"
+          />
+        </TableCell>
+      </TableRow>
```

Comment:
I am unsure if this is the correct place to put the Disable Tooltip in the checkbox. I noticed that the actual `SpatialTooltipSubscriber` component is rendered, unless the tooltip is disabled and decided to follow similar logic.

Reply:
This placement looks fine to me